source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2726,6 +2724,49 @@ export class Container extends CardElement {
         return element;
     }
 
+    protected truncateOverflow(maxHeight: number): boolean {
+        // Add 1 to account for rounding differences between browsers
+        var boundary = this.renderedElement.offsetTop + maxHeight + 1;
+        this.handleBottomOverflow(boundary);
```

Comment:
It doesn't look like handleBottomOverflow is called from any other place. As I was reading your changes, this indirection somewhat confused me. If this is indeed the only call site, let's move the code in handleBottomOverflow into truncateOverflow.